### PR TITLE
Remove k8s.io/kubernetes dependency for yurt-tunnel

### LINKED ISSUE #453: Eliminate dependency on k8s.io/kubernetes
https://github.com/kubernetes/kubernetes#to-start-using-k8s

> Use of the k8s.io/kubernetes module or k8s.io/kubernetes/... packages as libraries is not supported.

Relying directly on `k8s.io/kubernetes` didn't seem like a good idea, which made our project difficult to upgrade and maintain. I propose that eliminate dependency on `k8s.io/kubernetes`. 

This is likely to be a long-term task.

### PR DESCRIPTION
**Pull request by JcJinChen**
#### What this PR does / why we need it:
From kubernetes introduction: [ttps://github.com/kubernetes/kubernetes#to-start-using-k8s](https://github.com/kubernetes/kubernetes#to-start-using-k8s)

#### Which issue(s) this PR fixes:

Fixes #453

#### Does this PR introduce a user-facing change?

```
none
```

#### other Note

```
none
```

### CODE REVIEW

**Comment by rambohe-ch** [comment]
@JcJinChen maybe we need keep `iptables` from `k8s.io/kuberentes` to locate in a util directory.

**Comment by JcJinChen** [comment]
> @JcJinChen maybe we need keep `iptables` from `k8s.io/kuberentes` to locate in a util directory.

ok, i will submit again after modification.

**Comment by rambohe-ch** [comment]
@JcJinChen please fix license check and lint error.

File: pkg/yurthub/network/dummyif_test.go [hunk 1]
```diff
@@ -1,3 +1,4 @@
+//go:build linux
```
Comment (rambohe-ch) [inline pkg/yurthub/network/dummyif_test.go:1]: why add this comment?

File: pkg/yurthub/network/dummyif_unsupported.go [hunk 2]
```diff
@@ -1,3 +1,4 @@
+//go:build !linux
```
Comment (rambohe-ch) [inline pkg/yurthub/network/dummyif_unsupported.go:1]: the same as above comment

File: go.mod [hunk 3]
```diff
@@ -24,6 +24,7 @@ require (
 	github.com/stretchr/testify v1.4.0
 	github.com/vishvananda/netlink v1.0.0
 	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d // indirect
+	golang.org/x/sys v0.0.0-20200519105757-fe76b779f299
```
Comment (rambohe-ch) [inline go.mod:27]: why add this package?

File: pkg/yurthub/network/dummyif_test.go (hunk 1 above)
Comment (JcJinChen) [reply pkg/yurthub/network/dummyif_test.go:1]: After coding, to format my code, so I execute gofmt -w .,then there is an extra line(go:build linux) in some files located under the path pkg/uitl/iptables. But I found that there is no such line in the kubernetes codekubernetes, so i delete this extre line(go:build linux) in those files, forgot to delete this line in pkg/yurthub/network/dummyif_test.go. I think this is caused by gofmt.

**Review by rambohe-ch — APPROVED** [review]

**Pull request merged**